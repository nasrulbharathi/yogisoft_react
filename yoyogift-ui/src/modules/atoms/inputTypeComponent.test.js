import React from "react";
import { shallow } from "enzyme";
import InputTypeComponent from "./InputTypeComponent";
import { Input } from "@material-ui/core";
import { InputAdornment } from "@material-ui/core";

const response = () => Promise.resolve("");

describe("InputTypeComponent", () => {
  const props = {
    fieldFullWidth: "",
    inputError: "",
    styles: {},
    inputDisabled: false,
    inputId: 1,
    inputName: "text",
    inputType: "text",
    inputValue: "",
    handleInputChange: jest.fn(),
    inputPlaceholder: "Placehoder",
    endAdornment: false,
    startAdornment: false,
    inputHelperText: "",
  };
  it("should be defined", () => {
    expect(InputTypeComponent).toBeDefined();
  });
  it("should render InputTypeComponent", () => {
    const component = shallow(<InputTypeComponent {...props} />);
    expect(component).toMatchSnapshot();
  });
  it("should check for onChange method (2)", () => {
    const component = shallow(<InputTypeComponent {...props} />);
    component.find(Input).at(0).props().onChange();
    expect(props.handleInputChange).toBeCalled();
  });
});
