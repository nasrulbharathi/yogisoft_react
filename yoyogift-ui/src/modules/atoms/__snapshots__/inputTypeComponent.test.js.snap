// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputTypeComponent should render InputTypeComponent 1`] = `ShallowWrapper {}`;
